lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,446 @@
+import 'dart:math';
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class CreateBuffersAroundPoints extends StatefulWidget {
+  const CreateBuffersAroundPoints({super.key});
+
+  @override
+  State<CreateBuffersAroundPoints> createState() =>
+      _CreateBuffersAroundPointsState();
+}
+
+class _CreateBuffersAroundPointsState extends State<CreateBuffersAroundPoints>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  // A polygon that represents the valid area of use for the spatial reference.
+  late Polygon _boundaryPolygon;
+
+  // List of tap points.
+  final _bufferPoints = <PointAndRadius>[];
+
+  // Current status of the buffer.
+  var _status = Status.addPoints;
+
+  // Buffer radius.
+  var _bufferRadius = 100.0;
+
+  // Union status.
+  var _shouldUnion = false;
+
+  // Define the graphics overlays.
+  final bufferGraphicsOverlay = GraphicsOverlay();
+  final tapPointGraphicsOverlay = GraphicsOverlay();
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        showModalBottomSheet(
```

Comment:
I don't think we want the bottom sheet to be modal? Usually on our samples you should be able to navigate and do taps on the map even when the Settings sheet is up.

Reply:
Got it removed the `showModalBottomSheet`. Now setting sheet is up even when we tap on the map.